csrc/mmdeploy/backend_ops/tensorrt/batched_bev_nms/trt_batched_bev_nms.cpp

```diff
@@ -0,0 +1,235 @@
+// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Is this ops Rotated NMS with index output?
Why don't we just update Rotated NMS instead of create a new op?

Reply:
Yes. 
`Casue I found nms and nms_rotate reuse the same cuda kernel function. So I follow their implement style. And I'm worried that modifying nms_rotated will cause the MMrotate deployment error.

Reply:
I see, we can keep this op in this PR and merge it with rotated nms in the future.

Reply:
:relaxed: